CONTRIBUTING.md

```diff
@@ -15,13 +15,13 @@ Welcome to [TheAlgorithms/Go](https://github.com/TheAlgorithms/Go)! Before submi
 Being a contributor at The Algorithms, we request you to follow the points mentioned below:
 
 - You did your own work.
-    - No plagiarism allowed. Any plagiarized work will not be merged.
+- No plagiarism is allowed. Any plagiarized work will not be merged.
```

Comment:
I believe that this indentation was deliberate.

Reply:
What for?

Reply:
@cclauss I just used CPP contributing guidelines for our repository so, you're right and it should have the indentation 
https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md 

Should I change it?